mainapp/templates/mainapp/contributor_form.html

```diff
@@ -12,8 +12,8 @@
       Branch: City Branch, TVM<br>
       IFS Code: SBIN0070028<br>
       <br>
-      UPI 1: cmdrf@fbl<br>
-      UPI 2: keralacmdrf@sib<br>
+      UPI 1: <a href="upi://pay?pa=cmdrf@fbl&pn=Chief%20Minister%27s%20Distress%20Relief%20Fund">cmdrf@fbl</a><br>
+      UPI 2: <a href="upi://pay?pa=keralacmdrf@sib&pn=Chief%20Minister%27s%20Distress%20Relief%20Fund">keralacmdrf@sib</a><br>
```

Comment:
Please add `keralacmdrf@sbi` also.

Reply:
Isn't it better to keep the options short. It might lead to extra confusion for the users.

Reply:
Only keralacmdrf@sbi is working with UPI apps. Rest of them are getting detected as invalid UPI id (Tested with Tez, BHIM, PhonePe & PayTM) .

Reply:
Links are working with Chrome and is opening on UPI apps correctly.

Reply:
Can somebody also add the consolidated qr codes as image to the repo ? That can then be referred to later on.